agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4408,10 +4409,58 @@ bool slave_thread::handle_client_steering_request(Socket *sd, ieee1905_1::CmduMe
     }
 }
 
-
-bool slave_thread::handle_beacon_metrics_query_request(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
+bool slave_thread::handle_beacon_metrics_query_request(Socket *sd,
+                                                       ieee1905_1::CmduMessageRx &cmdu_rx)
```

Comment:
Did you run `clang_format`?

Reply:
I run it fro time to time, not necessarily with each commit.
I think we should consider adding clang-format _per compilation_.
i.e. automatically format only the files that were changed.
I guess currently it is not done this way since formatting and compiling are done in two different environments.